agendas/2021-05-13.md

```diff
@@ -91,5 +94,4 @@ Example agenda item:
    - Have since updated the implementation to more closely mirror the spec.
    - Potential validation oversight - @skip/@include; see related RFC: https://github.com/graphql/graphql-spec/pull/860
    - Question: have other implementors had a chance to look at this yet?
-1. [Status update on the Typescript migration](https://github.com/graphql/graphql-js/issues/2104)
```

Comment:
@Urigo I moved it up the list since it related to the GraphQL spec release, hope you don't mind.

Reply:
@IvanGoncharov sure, I just hope you can still give an update specifically for for the Typescript migration